docs/kubernetes-apps/local_volume_provisioner.md

```diff
@@ -1,10 +1,11 @@
-# Local Storage Provisioner
+# Local Static Storage Provisioner
```

Comment:
```suggestion
# Local Persistence Volume Static Provisioner
```
as the title of https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner